README.md

```diff
@@ -0,0 +1,77 @@
+# looking-glass
+
+Looking Glass is a tool for mirroring objects to Artifactory.  Currently only S3 is supported as a source, but FTP support will be added in the near future.
+
+## Why would you want to do this?
+
+It is common for vendors to release binaries to an S3 bucket or FTP.  To prevent 100's of servers from reaching outside our network during deployments, we had manually copied these binaries to our Artifactory instance.  Looking Glass automates this for us, so we know our Artifactory repo is always up to date! 
+
+
+# Setup
```

Comment:
I assume we are going to push the artifacts for this to the github releases page? should we mention that in the readme?

Reply:
Correct, it will push to github releases.  I'll update the readme to note that.